### PR TITLE
Create .settings.yml

### LINKED ISSUE #479: Labels needed for different features/areas
- [ ] additional standard labels from core repo
- [ ] create with *.yml file

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Labels have been imported now from owncloud/core

Fixes #479

### CODE REVIEW

File: .github/.settings.yml [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    color: ededed
+  - name: 2 - Developing
+    color: ededed
+  - name: 3 - To Review
+    color: ededed
+  - name: 4 - To release
+    color: ededed
+  - name: Acceptance tests
+    color: ededed
+  - name: CalDAV
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    color: c2e0c6
+  - name: 'Effort: 2d'
+    color: c2e0c6
+  - name: 'Effort: 2w'
+    color: c2e0c6
+  - name: 'Effort: 4d'
+    color: c2e0c6
+  - name: 'Effort: trivial'
+    color: c2e0c6
+  - name: Hacktoberfest
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+    color: 8c8cea
+  - name: Waiting
+    color: e6e6e6
+  - name: Waiting Internal
+    color: e6e6e8
+  - name: Waiting On Review/Feedback
+    color: ededed
+  - name: app:dav
+    color: 02d7e1
+  - name: app:encryption
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+    color: 02d7e1
+  - name: app:provisioning_api
+    color: 02d7e1
+  - name: app:search
+    color: 02d7e1
+  - name: app:user_ldap
+    color: 02d7e1
+  - name: app:windows_network_drive
+    color: ededed
+  - name: backport-request
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml (hunk 3 above)
Comment (DeepDiver1975) [reply .github/.settings.yml]: basicall all app labels

File: .github/.settings.yml [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+    color: 006b75
+  - name: blue-ticket
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+    color: 0033CC
+  - name: bounty
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+    color: 207de5
+  - name: bug
+    color: fc2929
+  - name: bug-analysis
+    color: fbca04
+  - name: bug-solved
+    color: 0e8a16
+  - name: clarification request
+    color: d4c5f9
+  - name: comp:apps
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun: all comp:

File: .github/.settings.yml [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+    color: d4c5f9
+  - name: comp:core-js
+    color: d4c5f9
+  - name: comp:filesystem
+    color: d4c5f9
+  - name: core-bug
+    color: ededed
+  - name: credits
+    color: e58729
+  - name: db:mariadb
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun: - all db:

File: .github/.settings.yml [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+    color: '444444'
+  - name: db:sqlite
+    color: '444444'
+  - name: dependencies
+    color: 0025ff
+  - name: design
+    color: cc317c
+  - name: dev:acceptance-tests
+    color: ff9966
+  - name: dev:appframework
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

Comment (DeepDiver1975) [reply .github/.settings.yml]: all dev - these are old and unused

File: .github/.settings.yml [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+    color: FF9966
+  - name: discussion
+    color: fbca04
+  - name: documentation
+    color: BFDADC
+  - name: enhancement
+    color: 02e10c
+  - name: enhancement-PM
+    color: FF9900
+  - name: feature:activity
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:  all feature:

File: .github/.settings.yml [hunk 11]
```diff
@@ -0,0 +1,300 @@
...
+    color: d4c5f9
+  - name: feature:sidebar
+    color: d4c5f9
+  - name: feature:tags
+    color: d4c5f9
+  - name: feature:theming
+    color: d4c5f9
+  - name: feature:update
+    color: d4c5f9
+  - name: gitmate-never-close
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 12]
```diff
@@ -0,0 +1,300 @@
...
+    color: d4c5f9
+  - name: in progress
+    color: ededed
+  - name: invalid
+    color: ededed
+  - name: javascript
+    color: '168700'
+  - name: jenkins
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 13]
```diff
@@ -0,0 +1,300 @@
...
+    color: '996633'
+  - name: junior job
+    color: f6d300
+  - name: mobile:Android
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 14]
```diff
@@ -0,0 +1,300 @@
...
+    color: 207de5
+  - name: mobile:iOS
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 15]
```diff
@@ -0,0 +1,300 @@
...
+    color: 207de5
+  - name: needs info
+    color: 0b02e1
+  - name: nginx
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 16]
```diff
@@ -0,0 +1,300 @@
...
+    color: 5319e7
+  - name: old-inactive
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 17]
```diff
@@ -0,0 +1,300 @@
...
+    color: fc2929
+  - name: p3-medium
+    color: fc2929
+  - name: p4-low
+    color: fc2929
+  - name: packaging
+    color: F7C6C7
+  - name: performance
+    color: fbca04
+  - name: php
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 18]
```diff
@@ -0,0 +1,300 @@
...
+    color: 45229e
+  - name: php:7.2
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 19]
```diff
@@ -0,0 +1,300 @@
...
+    color: 006b75
+  - name: purple-ticket
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 20]
```diff
@@ -0,0 +1,300 @@
...
+    color: e11d21
+  - name: release note
+    color: BFD4F2
+  - name: research
+    color: FFFF00
+  - name: review carefully!
+    color: ededed
+  - name: security
+    color: e11d21
+  - name: server
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 21]
```diff
@@ -0,0 +1,300 @@
...
+    color: ededed
+  - name: server-sprint
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml [hunk 22]
```diff
@@ -0,0 +1,300 @@
...
+    color: ededed
+  - name: settings:admin
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun: all settings:

File: .github/.settings.yml [hunk 23]
```diff
@@ -0,0 +1,300 @@
...
+    color: 02d7e1
+  - name: settings:installation
+    color: 02d7e1
+  - name: settings:login
+    color: 02d7e1
+  - name: settings:personal
+    color: 02d7e1
+  - name: settings:users
+    color: 02d7e1
+  - name: sev1-critical
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun: - prios sould be enough for us

File: .github/.settings.yml [hunk 24]
```diff
@@ -0,0 +1,300 @@
...
+    color: fc2929
+  - name: sev4-low
+    color: fc2929
+  - name: status/STALE
+    color: ededed
+  - name: technical debt
+    color: FFFF00
+  - name: tested
+    color: FBCA04
+  - name: v11
```
Comment (DeepDiver1975) [inline .github/.settings.yml]: :gun:

File: .github/.settings.yml (hunk 10 above)
Comment (michaelstingl) [reply .github/.settings.yml]: Hm, I'd love to have some `feature:…` labels to categorize issues. What would be the alternative?

Comment (DeepDiver1975) [reply .github/.settings.yml]: I'm happy to add feature labels once we have some - as of today everything is files from my understansing. Furthermore the feature labels in core are necessary to manage the masses of thousands of issue which have been opened in there for various of apps.

**Pull request merged**